datahub-web-react/src/app/ingest/source/builder/RecipeForm/utils.ts

```diff
@@ -157,6 +256,30 @@ export const STATEFUL_INGESTION_ENABLED: RecipeField = {
     rules: null,
 };
 
+export const UPSTREAM_LINEAGE_IN_REPORT: RecipeField = {
+    name: 'upstream_lineage_in_report',
+    label: 'Include Upstream Lineage In Report.',
+    tooltip: 'Useful for debugging lineage information. Set to True to see the raw lineage created internally.',
+    type: FieldType.BOOLEAN,
+    fieldPath: 'source.config.upstream_lineage_in_report',
+    rules: null,
+};
+
+export const TABLE_LINEAGE_MODE: RecipeField = {
+    name: 'table_lineage_mode',
+    label: 'Table Lineage Mode',
+    tooltip:
+        'Which table lineage collector mode to use. Check out the documentation explaining the difference between the three available modes." where "the documentation" is a link to https://datahubproject.io/docs/generated/ingestion/sources/redshift/#lineage',
```

Comment:
lol sorry that last sentence is describing the rest of the tooltip. So like the tooltip should be:
`Which table lineage collector mode to use. Check out the documentation explaining the difference between the three available modes.`
And the words "the documentation" should be a link to the url so that users can click the link to go check out documentation